# Generated by Django 4.2.7 on 2023-12-27 16:22

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0003_user_biography_user_facebook_user_instagram_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(default=None, error_messages={'unique': 'Пользователь с таким именем пользователя уже существует.'}, help_text='Обязательное поле. Не более 30 символов. Только буквы, цифры и @/./+/-/_.', max_length=30, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()]),
        ),
    ]
